### PR TITLE
doc: -lua-sources

### PR DESCRIPTION
**Pull request by szuecs**
doc: -lua-sources

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

File: docs/reference/scripts.md [hunk 1]
```diff
@@ -49,6 +49,17 @@ Lua `print` builtin function writes skipper info log messages.
...
+## Enable and Disable lua sources
+
+The flag `-lua-sources` allows to set 5 different values:
+
+* "file": Allows to use reference to file for scripts
+* "inline": Allows to use inline scripts
+* "inline", "file": Allows to use reference to file and inline scripts
+* "none": Disable Lua filters
+* "": the same as "inline", "file", the default value for binary and
+  library users
```
Comment (RomanZavodskikh) [inline docs/reference/scripts.md:61]: "inline" for binary and "file" for library?
In this case I think we need to add "respectively" to the end of this sentence.

File: docs/reference/scripts.md [hunk 2]
```diff
@@ -49,6 +49,17 @@ Lua `print` builtin function writes skipper info log messages.
 
 `sleep(number)` function pauses execution for at least `number` milliseconds. A negative or zero duration causes `sleep` to return immediately.
 
+## Enable and Disable lua sources
+
+The flag `-lua-sources` allows to set 5 different values:
+
+* "file": Allows to use reference to file for scripts
+* "inline": Allows to use inline scripts
+* "inline", "file": Allows to use reference to file and inline scripts
```
Comment (AlexanderYastrebov) [inline docs/reference/scripts.md:58]: `"inline,file"`

**Comment by AlexanderYastrebov** [comment]
:+1:

File: docs/reference/scripts.md (hunk 1 above)
Comment (szuecs) [reply docs/reference/scripts.md:61]: No both are for users that create routes (our endusers), but skipper binary or skipper library users are the ones running the system.

File: docs/reference/scripts.md (hunk 2 above)
Comment (szuecs) [reply docs/reference/scripts.md:58]: Because it's a listFlag and no split string I did this on purpose

**Pull request merged**